homeassistant/components/hdmi_cec/__init__.py

```diff
@@ -383,6 +405,9 @@ def update(self):
     async def async_added_to_hass(self):
         """Register HDMI callbacks after initialization."""
         self._device.set_update_callback(self._update)
+        self.hass.bus.async_listen(
```

Comment:
We should not use the event bus for intra integration communication. We have our dispatch helper for that purpose. Please use that instead.